src/test/regress/sql/explain_analyze.sql

```diff
@@ -1,6 +1,10 @@
 -- start_matchsubs
--- m/ Gather Motion [12]:1  \(slice1; segments: [12]\)/
--- s/ Gather Motion [12]:1  \(slice1; segments: [12]\)/ Gather Motion XXX/
+-- m/\(actual time=\d+\.\d+..\d+\.\d+ rows=\d+ loops=\d+\)/
+-- s/\(actual time=\d+\.\d+..\d+\.\d+ rows=\d+ loops=\d+\)/(actual time=##.###..##.### rows=# loops=#)/
+-- m/Gather Motion \d+:\d+\s+\(slice\d+; segments: \d+\)/
+-- s/Gather Motion \d+:\d+\s+\(slice\d+; segments: \d+\)/Gather Motion #:#  \(slice#; segments: #\)/
+-- m/\(slice\d+\)\s+Executor memory: \d+[KMG] bytes(?:.*?)(?=\n)/
+-- s/\(slice\d+\)\s+Executor memory: \d+[KMG] bytes(?:.*?)(?=\n)/\(slice#) Executor memory: # bytes ####/
```

Comment:
Can we add to matchsubs this lines only without changing anything else?
```
-- m/\(actual time=\d+\.\d+..\d+\.\d+ rows=\d+ loops=\d+\)/
-- s/\(actual time=\d+\.\d+..\d+\.\d+ rows=\d+ loops=\d+\)/(actual time=##.###..##.### rows=# loops=#)/
```

Reply:
Fixed, adjusted the tests to not edit matchsubs.